README.md

```diff
@@ -43,47 +43,45 @@ DTOs, and populate services.  the PopulateCliApp can be used to populate specifi
-       * set environment variable: `B2C_TENANT_NAME=ddpdevb2c`
-       * set environment variable: `B2C_CLIENT_ID=<<vault read -field value secret/dsp/ddp/b2c/dev/application_id>>`
-       * set environment variable: `B2C_POLICY_NAME=B2C_1A_ddp_participant_signup_signin_dev`
-       * set environment variable: `DSM_JWT_SIGNING_SECRET=<<vault read -field jwt_signing_secret secret/dsp/ddp/d2p/dev/dsm>>`
-       * set environment variable: `SENDGRID_API_KEY=<<vault read -field=api_key secret/dsp/ddp/d2p/dev/sendgrid>>`
+       * Set the Active profiles field to: `human-readable-logging, development`
+       * Disable launch optimization by clicking `Modify options > Disable launch optimization`
+       * Render environment variables with `bash ./local-dev/render_environment_vars.sh ApiAdminApp <YOUR_EMAIL>`
+         * The output should be a semicolon-separated list of environment variables.
+         * Copy this output into the "Environment variables" field of the run configuration. (Click `Modify options > Environment variables` if this is not visible)
+       * Your final Run Configuration should look similar to this:
+        * <img width="840" alt="Admin API run configuration" src="https://github.com/broadinstitute/juniper/assets/7257391/ae82332c-628f-40e1-8917-b7536af1eb02">
      * ApiParticipantApp (in api-participant module)
-        * set active profiles of "human-readable-logging" and "development"
-        * set environment variable: `SENDGRID_API_KEY=<<vault read -field=api_key secret/dsp/ddp/d2p/dev/sendgrid>>`
-        * set environment variable: `REDIRECT_ALL_EMAILS_TO=<<your email address>>`
-        * set environment variable: `B2C_TENANT_NAME=ddpdevb2c`
-        * set environment variable: `B2C_CLIENT_ID=<<vault read -field value secret/dsp/ddp/b2c/dev/application_id>>`
-        * set environment variable: `B2C_POLICY_NAME=B2C_1A_ddp_participant_signup_signin_dev`
-        * set environment variable: `B2C_CHANGE_PASSWORD_POLICY_NAME=B2C_1A_ddp_participant_change_password_dev`
-        * disable launch optimization
-        
-         
+        * Set the Active profiles field to: `human-readable-logging, development`
+        * Disable launch optimization by clicking `Modify options > Disable launch optimization`
+        * Render environment variables with `bash ./local-dev/render_environment_vars.sh ApiParticipantApp <YOUR_EMAIL>`
+          * The output should be a semicolon-separated list of environment variables.
+          * Copy this output into the "Environment variables" field of the run configuration. (Click `Modify options > Environment variables` if this is not visible)
+        * Your final Run Configuration should look similar to this:
+        * <img width="840" alt="Participant API run configuration" src="https://github.com/broadinstitute/juniper/assets/7257391/b8e006fe-65a7-4731-9ab1-bda8d38b480b">
```

Comment:
How do things get into `juniper/assets`?

Reply:
If you edit a file in the GitHub UI, you can drag/drop files into the editor and it adds them to `assets`

![Screenshot 2023-09-12 at 2 59 35 PM](https://github.com/broadinstitute/juniper/assets/7257391/b54800a6-2c03-412e-8ac2-2173926fb15b)